VISUALIZERS/DATA_STRUCTURE/TEXT_VIEW/TEXT_VIEW.py

```diff
@@ -4,7 +4,17 @@
 @flojoy
 def TEXT_VIEW(default: TextBlob):
     """
-    The TEXT_VIEW node creates a text visualization for a given TextBlob DC type.
+    The TEXT_VIEW node creates a text visualization for a given TextBlob DataContainer type.
+
+    Inputs
+    ------
+    default : TextBlob
+        DataContainer object to be visualized in text format
```

Comment:
"Text data to be visualized"

Reply:
It looks like the information might be a bit misleading. The input doesn't actually accept regular text data; it takes text data in the form of a DataContainer. So, I think it would be better to say, "the DataContainer to be visualized in text format." Also, we've already specified the type of DataContainer that this node will accept.